doc/man3/EVP_PKEY_size.pod

```diff
@@ -25,10 +25,10 @@ L<EVP_PKEY_encrypt(3)>, L<EVP_PKEY_decrypt(3)>, L<EVP_PKEY_derive(3)>.
 It must be stressed that, unless the documentation for the operation
 that's being performed says otherwise, the size returned by
 EVP_PKEY_size() is only preliminary and not exact, so the final
-contents of the target buffer may be smaller.  It is therefore crucial
-to take note of the size given back by the function that performs the
-operation, such as L<EVP_PKEY_sign(3)> (the I<siglen> argument will
-receive that length), to avoid bugs.
+contents of the target buffer may be smaller or larger than indicated.
+It is therefore crucial to take note of the size given back by the
+function that performs the operation, such as L<EVP_PKEY_sign(3)>
+(the I<siglen> argument will receive that length), to avoid bugs.
```

Comment:
I am not sure this is right change. The first paragraph is completely invalidated by this change. Which algorithms produce signatures bigger than EVP_PKEY_size? And shouldn't the fix be to actually change the EVP_PKEY_size to return something to conform to its documentation?

Reply:
No! They cannot be larger. Must be equal to or smaller than. Not larger.